### DISCUSSION TITLE
Adding Captcha field?

### DISCUSSION THREAD

**Discussion by ajtatum** [ohmyform/ohmyform#231] [Ideas]
I really like OhMyForm, but I'm hesitant to put it online as there's no captcha or any spam prevention that I see. It would be great if it could support Google ReCaptcha or Cloudflare Turnstile or something similar. Are there any plans for this? Is there a way to work this into the existing install that I'm not seeing?

**Comment by wodka** [comment]
hi, are you thinking about doing this when the user submits a form right? Not yet planned in any way but I think this could be interesting.

**Reply by ajtatum** [reply]
Yes, that's correct.
